### PR TITLE
Easy-1804 set state of a deposit to uploaded, once all files are uploaded to the sword2 directory

### PR DESCRIPTION
**Pull request by stephanHogenboom**
Fixes EASY-Easy 1804 uploaded
#### When applied it will
* will give states the status uploaded instead of finalizing once the file is uploaded

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#7_deposit-lib](https://github.com/DANS-KNAW/dans-deposit-lib/pull/7)     | ..
easy-                      | [EMD_PR#36](https://github.com/DANS-KNAW/easy-manage-deposit/pull/36)     | ..

### CODE REVIEW

File: src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala [hunk 1]
```diff
@@ -469,13 +469,10 @@ object DepositHandler {
   }
 
   def isOnPosixFileSystem(file: File): Boolean = {
-    try {
+    Try {
```
Comment (rvanheest) [inline src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala]: good catch! That's better indeed.
**challenge**: you could even make this into a one-liner if you want. hint: use [Try.fold](https://www.scala-lang.org/api/current/scala/util/Try.html#fold[U](fa:Throwable=%3EU,fb:T=%3EU):U)

**Review by rvanheest — APPROVED** [review]

File: src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala (hunk 1 above)
Comment (stephanHogenboom) [reply src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala]: `Try(Files.getPosixFilePermissions(file.toPath)).fold(_ => false, _ => true)?`

Comment (rvanheest) [reply src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala]: you rock! 👍

Comment (stephanHogenboom) [reply src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala]: Thanks for the tip!

**Pull request merged**